core/object/class_db.cpp

```diff
@@ -373,6 +373,19 @@ bool ClassDB::can_instantiate(const StringName &p_class) {
 	return (!ti->disabled && ti->creation_func != nullptr && !(ti->gdextension && !ti->gdextension->create_instance));
 }
 
+bool ClassDB::is_abstract(const StringName &p_class) {
+	OBJTYPE_RLOCK;
+
+	ClassInfo *ti = classes.getptr(p_class);
+	if (!ti) {
+		if (!ScriptServer::is_global_class(p_class)) {
+			ERR_FAIL_V_MSG(false, "Cannot get class '" + String(p_class) + "'.");
+		}
+		return false;
+	}
+	return (ti->creation_func == nullptr && (!ti->gdextension || ti->gdextension->create_instance == nullptr));
```

Comment:
Shouldn't this also test `ti->disabled` like in `can_instantiate()`?

Reply:
Probably, if a class is disabled, it does not cease to be abstract. That is, whether the class is enabled is not important for `is_abstract()`, but it is important for `can_instantiate()`.